protostar_cairo/cairo_project.toml

```diff
@@ -0,0 +1,2 @@
+[crate_roots]
```

Comment:
Whatr is the purpose of this package?

Reply:
Our configuration object passes this as part of project's `cairo-path`

Reply:
Would be nice to remove it from code instead, please do that if it is not too complex